app/views/studies/_expression_file_info_fields.html.erb

```diff
@@ -25,9 +25,9 @@
                  {}, class: 'form-control' %>
```

Comment:
library_preparation_protocol should not have a default value, can we add `include_blank: true` so the default is blank?

Reply:
The field is marked as required, so I've added a prompt to show "Select one..." by default.